dxr/app.py

```diff
@@ -162,6 +164,30 @@ def _tree_tuples(trees, tree, query_text, is_case_sensitive):
              description)
             for t, description in trees.iteritems()]
 
+@dxr_blueprint.route('/<tree>/images/<path:path>')
+def send_image(tree, path):
+    '''
```

Comment:
Just a nit, but we'll obviously want to stay consistent with the docstring quoting in the rest of the project.